spec/spec_helper.rb

```diff
@@ -8,3 +8,5 @@
 Dir[File.join(File.dirname(__FILE__), 'support', '**/*.rb')].each do |file|
   require file
 end
+
+ENV['IEX_CLOUD_SECRET_TOKEN'] = 'test-iex-cloud-secret-token'
```

Comment:
Make this `ENV[...] ||=` 

If one needs to record a new VCR I can then can do it without modifying code with `IEX_CLOUD_SECRET_TOKEN=... rspec spec/...`.